packaging/examples/security/scram-sha-512-auth/mirror-maker-2.yaml

```diff
@@ -202,7 +202,7 @@ spec:
           name: "*"
         operations:
           - Read
-      # MirrorHeartbeatConnector
+      # MirrorHeartbeatConnector => needed only if the connector is enabled
```

Comment:
Should we remove that too?
Same below

Reply:
I would not remove it -> I think it is useful to track this. But we can for example consider commenting out the ACLs.